tests/Aspire.Hosting.Oracle.Tests/AddOracleTests.cs

```diff
@@ -18,7 +19,7 @@ public void AddOracleAddsGeneratedPasswordParameterWithUserSecretsParameterDefau
 
         var orcl = appBuilder.AddOracle("orcl");
 
-        Assert.IsType<UserSecretsParameterDefault>(orcl.Resource.PasswordParameter.Default);
+        Assert.Equal(nameof(UserSecretsParameterDefault), orcl.Resource.PasswordParameter.Default?.GetType().Name);
```

Comment:
Use strings instead so we don't need IVT or a shared source file.

https://github.com/dotnet/aspire/blob/c6ec314738bb762a022869f6c70176933fe41556/tests/Aspire.Hosting.Qdrant.Tests/AddQdrantTests.cs#L25